src/panels/config/scene/ha-scene-editor.ts

```diff
@@ -0,0 +1,727 @@
+                  html`
+                    <ha-card>
+                      <div class="card-header">
+                        ${device.name}
+                        <paper-icon-button
+                          icon="hass:delete"
+                          title="${this.hass.localize(
+                            "ui.panel.config.scene.editor.devices.delete"
+                          )}"
+                          .device=${device.id}
+                          @click=${this._deleteDevice}
+                        ></paper-icon-button>
+                      </div>
+                      ${device.entities.map((entity) => {
+                        const stateObj = this.hass.states[entity];
+                        if (!stateObj) {
+                          return html``;
+                        }
+                        return html`
+                          <paper-icon-item
+                            .entity=${stateObj.entity_id}
+                            @click=${this._showMoreInfo}
+                            class="device-entity"
+                          >
+                            <state-badge
+                              .stateObj=${stateObj}
+                              slot="item-icon"
+                            ></state-badge>
+                            <paper-item-body>
+                              ${computeStateName(stateObj)}
+                            </paper-item-body>
+                          </paper-icon-item>
+                        `;
+                      })}
+                    </ha-card>
+                  `
+              )}
+
+              <ha-card
+                header=${this.hass.localize(
+                  "ui.panel.config.scene.editor.devices.add"
+                )}
+              >
+                <div class="card-content">
+                  <ha-device-picker
+                    @value-changed=${this._devicePicked}
+                    .hass=${this.hass}
+                    label=${this.hass.localize(
+                      "ui.panel.config.scene.editor.devices.add"
+                    )}
+                  />
+                </div>
+              </ha-card>
+            </ha-config-section>
+
+            ${this.showAdvanced
+              ? html`
+                  <ha-config-section .isWide=${!this.narrow}>
+                    <div slot="header">
+                      ${this.hass.localize(
+                        "ui.panel.config.scene.editor.entities.header"
+                      )}
+                    </div>
+                    <div slot="introduction">
+                      ${this.hass.localize(
+                        "ui.panel.config.scene.editor.entities.introduction"
+                      )}
+                    </div>
+                    ${entities.length
+                      ? html`
+                          <ha-card
+                            class="entities"
+                            header=${this.hass.localize(
```

Comment:
```suggestion
                            .header=${this.hass.localize(
```

Reply:
I thought it only needed the `.` when the value would change?

Reply:
Properties are always faster, because with Lit, all attributes are first converted to properties, then compared to see if they should trigger a render. Attributes are only better if it's not dynamic at all, because in that case Lit just stamps out the template, and doesn't check if it needs updating each render.